server/src-exec/Ops.hs

```diff
@@ -241,6 +241,20 @@ from6To7 = liftTx $ do
     $(Q.sqlFromFile "src-rsr/migrate_from_6_to_7.sql")
   return ()
 
+from7To8
+  :: (MonadTx m, HasHttpManager m, CacheRWM m, UserInfoM m, MonadIO m)
+  => m ()
+from7To8 = do
+  -- migrate database
+  Q.Discard () <- liftTx $ Q.multiQE defaultTxErrorHandler
+    $(Q.sqlFromFile "src-rsr/migrate_from_7_to_8.sql")
+  -- migrate metadata
+  void $ runQueryM migrateMetadataFrom7
+  setAsSystemDefined
```

Comment:
`setAsSystemDefined` is too broad. Use a statement which will only make the modifications for the added metadata.